web_tree_image_tooltip/README.rst

```diff
@@ -0,0 +1,43 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :alt: License: AGPL-3
+
+=================================================
+Display images and icons in tree view via Tooltip
+=================================================
+
+This module defines a images and icons in tree view via Tooltip.
+Optionally, set a ``width`` attribute. Default width is 30px.
```

Comment:
This is false. I don't see the handling of `width` in the Javascript. 

Use the new split readme: https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

Reply:
> Use the new split readme: https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

@tarteo It's required?

Reply:
Don't know, but it looks much better.

Reply:
> Don't know, but it looks much better.

@tarteo Done, Yes it looks Good.